src/App.js

```diff
@@ -2,18 +2,54 @@ import React from 'react';
 import './App.css';
 
 import users from './api/users';
+import todos from './api/todos';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+import { TodoForm } from './Components/TodoForm';
+import { TodoList } from './Components/TodoList';
+
+const setTodos = todos.map(todo => ({
```

Comment:
это массив объектов, а не функция
```suggestion
const preparedTodos = todos.map(todo => ({
```